lib/fairy/fairy.c

```diff
@@ -27,6 +27,7 @@
 #include "macros.h"
 
 VerbosityLevel gVerbosity = VERBOSITY_NONE;
+bool gUseRealAlignment = false;
```

Comment:
I'm not sure about "real" here. How about `useElfAlignment`?

Reply:
Yeah, that sounds better. I wasn't sure how to name that variable